src/data/nav.yaml

```diff
@@ -115,6 +115,8 @@
               path: /docs/cli-command-reference/ecosystem-cli-runs-prepare
             - title: Running tests in the Ecosystem
               path: /docs/cli-command-reference/ecosystem-cli-runs-submit
+            - title: Retrying and deleting tests

```

Comment:
Please can we change 'deleting' to 'cancelling' just to avoid confusion with deleting records from the RAS

Reply:
Yes I see what you are saying - will do. Will also update in the other places I have used that term(s)